### PR TITLE
Door movers

### PR DESCRIPTION
**Pull request by chrisforbes**

### CODE REVIEW

File: assets/default.manifest [hunk 1]
```diff
@@ -110,6 +110,22 @@ assets = (
     offset = [0.5,0.0,0.0]
     nonconvex = true
 },
+{
+    name = "door-leftside"
+    type = "mesh"
+    file = "mesh/door-leftside.obj"
+    scale = 0.0625
```
Comment (RobotCaleb) [inline assets/default.manifest]: Density

File: assets/default.manifest [hunk 2]
```diff
@@ -110,6 +110,22 @@ assets = (
...
+    offset = [0.5,0.0,0.0]
+    nonconvex = true
+},
+{
+    name = "door-rightside"
+    type = "mesh"
+    file = "mesh/door-rightside.obj"
+    scale = 0.0625
```
Comment (RobotCaleb) [inline assets/default.manifest]: Density

File: nightmare.vcxproj [hunk 3]
```diff
@@ -87,7 +87,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='ReleaseWithDebug|x64'">
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
-      <Optimization>Disabled</Optimization>
+      <Optimization>MaxSpeed</Optimization>
```
Comment (RobotCaleb) [inline nightmare.vcxproj]: ?

File: gen/comp/door_slider [hunk 4]
```diff
@@ -0,0 +1,4 @@
+ui_name,Door slider
```
Comment (RobotCaleb) [inline gen/comp/door_slider]: door_slider or linear_actuator?

Comment (chrisforbes) [reply gen/comp/door_slider]: I'm not trying to make this generic -- it looks directly into a parent door component.

File: assets/default.manifest (hunk 2 above)
Comment (chrisforbes) [reply assets/default.manifest]: Fixed.

**Pull request merged**